opencga-app/app/scripts/azure/arm/solr/azuredeploy.json

```diff
@@ -280,5 +294,15 @@
         }
       }
     }
-  ]
+  ],
+  "outputs" : {
+    "solrConnectionDetails" : {
+      "value": "[if(equals(0, variables('zookeeperScaleNumber')), variables('solrURLs'), variables('zookeeperHosts'))]",
```

Comment:
Am I correct in thinking this will give the master if there we are replicated and fallback to the single Solr address if we're not?

Reply:
Ok just seen the PR #978